site/docs/hosting/deno-deploy.md

```diff
@@ -89,7 +89,7 @@ deployctl deploy --project <project> ./mod.ts --prod --token <token>
 After getting your app running, you should configure your bot's webhook settings to point to your app.
 To do that, send a request to
 
-```text
+```md
```

Comment:
I suggest reverting this since it doesn’t add any extra functionality except the unexpected coloring of `text` and `url`.

Reply:
I agree.